### PR TITLE
Mark packages as stable for 1.1.5

### PR DESCRIPTION
**Pull request by safern**
cc: @weshaggard

### CODE REVIEW

File: pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj [hunk 1]
```diff
@@ -70,7 +70,7 @@
       <Version>1.0.7</Version>
```
Comment (weshaggard) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: shouldn't this become 1.0.8?

File: pkg/projects/Microsoft.NETCore.App/project.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "Microsoft.DiaSymReader.Native": "1.4.1",
     "Libuv": "1.9.2",
     "runtime.native.System.IO.Compression": "4.3.1",
-    "runtime.native.System.Security.Cryptography": "4.3.2",
+    "runtime.native.System.Security.Cryptography": "4.3.3",
     "runtime.native.System.Security.Cryptography.Apple": "4.3.1",
     "runtime.native.System.Security.Cryptography.OpenSsl": "4.3.2"
```
Comment (weshaggard) [inline pkg/projects/Microsoft.NETCore.App/project.json]: Please be sure to check any of the new package versions we are shipping to ensure they are explicitly listed here (assuming they are in Microsoft.NETCore.App).

File: pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj (hunk 1 above)
Comment (safern) [reply pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: Yup you're right. Will update that.

**Comment by safern** [comment]
@weshaggard updated the version that needs to be fixed, and added OpenSsl in project.json and in Microsoft.NETCore.App.pkgproj for backward compatibility.

**Pull request merged**